### PR TITLE
Do not allocate event_meta_data when it is not needed

### PR DESCRIPTION
**Pull request by lutter**

### CODE REVIEW

File: store/postgres/src/entities.rs [hunk 1]
```diff
@@ -1103,7 +1126,12 @@ impl JsonStorage {
             None => return Ok(()),
             Some(event) => event,
         };
-
+        let event_id = match history_event.id {
+            Some(id) => id,
+            None => unreachable!(
+                "we always allocate an event_id when we need to record metadata history"
+            ),
+        };
```
Comment (leoyvens) [inline store/postgres/src/entities.rs]: This is an `expect`.

**Comment by leoyvens** [comment]
I'd rather pass this review to someone who knows how history events work, possibly @fordN or @Jannis.

File: store/postgres/src/entities.rs (hunk 1 above)
Comment (lutter) [reply store/postgres/src/entities.rs]: D'oh. Thanks for pointing that out. Fixed.

**Comment by lutter** [comment]
Rebased to latest master and addressed review comment.

**Pull request merged**